jupyter_sphinx/execute.py

```diff
@@ -123,12 +123,12 @@ def apply(self):
         linenos_config = self.config.jupyter_sphinx_linenos
         continue_linenos = self.config.jupyter_sphinx_continue_linenos
         # Check if we have anything to execute.
-        if not doctree.traverse(JupyterCellNode):
+        if not list(doctree.findall(JupyterCellNode)):
```

Comment:
Here and later converting to list looks OK, but I wonder if keeping an iterator is cleaner
```suggestion
        if not next(doctree.findall(JupyterCellNode), False):
```

Reply:
Updated